shop/tmp/post-install-scripts/post-install.sh

```diff
@@ -12,20 +12,22 @@ echo 'Czyszczenie bazy danych'
 SQL_SCRIPT_FILE="/var/sql/post-install.sql"
 
 
-if [ "$BIZ_USE_DUMP" -eq "1" ] ; then
+if [ "$BIZ_DB_USE_DUMP" -eq "1" ] ; then
 	echo 'Wykorzystywanie dumpa bazy danych'
-	SQL_SCRIPT_FILE="/var/sql/prestashop-dump.sql"
+	SQL_SCRIPT_FILE="/var/sql/prestashop_dump.sql"
 else
 	echo 'Wykorzystywanie skryptu inicjalizującego bazę'
 fi
 
 mysql -u$DB_USER -p$DB_PASSWD -h $DB_SERVER $DB_NAME < $SQL_SCRIPT_FILE
 
-if [ "$BIZ_USE_DUMP" -eq "1" ] ; then
+if [ "$BIZ_DB_USE_DUMP" -eq "1" ] ; then
 	echo 'Naprawa danych związanych z hostem w bazie danych'
-	envsubst '$PS_DOMAIN' < /var/ssl/sql-hostname-update.sql.tmpl > /var/ssl/sql-hostname-update.sql
+	envsubst '$PS_DOMAIN' < /var/sql/sql-hostname-update.sql.tmpl > /var/sql/sql-hostname-update.sql
 	# Po substytucji zmiennych środowiskowych wykonujemy to na serwerze
-	mysql -u$DB_USER -p$DB_PASSWD -h $DB_SERVER $DB_NAME < /var/ssl/sql-hostname-update.sql
+	mysql -u$DB_USER -p$DB_PASSWD -h $DB_SERVER $DB_NAME < /var/sql/sql-hostname-update.sql
+	# Odtworzenie zdjęć
+	unzip -q -o -u /tmp/img.zip -d /var/www/html/
```

Comment:
Zastanawia mnie dlaczego obrazki są w ZIPie, można je po prostu wrzucić jako volume i zrobić podlinkowanie do nich, np. link symboliczny z ln -s

Trzeba też ręcznie zbudować tego ZIPa, bo scraper robi to w katalogu. 

Katalog /tmp presta (chyba) wyczyści, ale mogą być w /var

Nie wiem po co je kompresować i potem dekompresować. Choć rozumiem, że mogą być tutaj kwestie o których nie pomyślałem.

Reply:
Obrazki w zipie dałem, bo ten katalog w sumie ma prawie 16000 obrazów i nie chciałem tylu rzeczy wrzucać na gita. I to nie są obrazy z samego scrapera, tylko dodane już przez api do prestashopa, a prestashop sobie jakoś specyficznie dorzuca te obrazki do folderów